MapboxCoreNavigation.podspec

```diff
@@ -40,9 +40,10 @@ Pod::Spec.new do |s|
   s.requires_arc = true
   s.module_name = "MapboxCoreNavigation"
 
-  s.dependency "MapboxNavigationNative", "~> 79.0"
-  s.dependency "MapboxDirections", "~> 2.0"
+  s.dependency "MapboxNavigationNative", "~> 80.0"
+  s.dependency "MapboxDirections-pre", ">= 2.1.0-rc.1", "< 3.0.0"
```

Comment:
I think previously we were pointing explicitly to a tagged version, without providing a range of accepted versions. But I guess such format should be fine as well.

Reply:
Yes, I was thinking about it. It was important for the 2.0 prerelease as we could have had breaking changes there. But now it should be fine. And in 2.1 GA we will have a regular `~> 2.1`.